gcp/table_gcp_storage_bucket_object.go

```diff
@@ -0,0 +1,349 @@
+package gcp
+
+import (
+	"context"
+	"strings"
+
+	"github.com/turbot/go-kit/helpers"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+	"google.golang.org/api/googleapi"
+	"google.golang.org/api/storage/v1"
+)
+
+func tableGcpBucketStorageObject(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_storage_bucket_object",
+		Description: "GCP Storage Object",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"bucket", "name"}),
+			Hydrate:    getBucketStorageObject,
+		},
+		List: &plugin.ListConfig{
```

Comment:
Are there any optional quals we can add that the API supports? We found these helpful in the `aws_s3_object` table to speed up queries

Reply:
@cbruno10 I tried setting `StartOffset` for an object, but it's not working as expected. No other optional quals are available for this API.